### PR TITLE
React to pipeline changes

### PR DESCRIPTION
**Pull request by pakrym**

### CODE REVIEW

File: src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsMessageParser.cs [hunk 1]
```diff
@@ -35,7 +36,7 @@ public ParseResult ParseMessage(ReadOnlyBuffer buffer, out Position consumed, ou
 
             while (buffer.Length > 0)
             {
-                if (ReadOnlyBuffer.Seek(start, end, out var lineEnd, ByteLF) == -1)
+                if (!(buffer.PositionOf(ByteLF) is SequencePosition lineEnd))
```
Comment (anurse) [inline src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsMessageParser.cs]: Maybe flip the conditional now that it's an `is` check?

Comment (pakrym) [reply src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsMessageParser.cs]: It makes a lot of code more nested

Comment (anurse) [reply src/Microsoft.AspNetCore.Sockets.Client.Http/ServerSentEventsMessageParser.cs]: Ok, fair enough. Just thought that if it were quick we should do it. Go ahead and merge without changing that.

File: src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/StreamExtensions.cs [hunk 2]
```diff
@@ -38,22 +38,22 @@ public static Task CopyToAsync(this Stream stream, IPipeWriter writer, Cancellat
         }
 
         /// <summary>
-        /// Copies a <see cref="ReadOnlyBuffer"/> to a <see cref="Stream"/> asynchronously
+        /// Copies a <see cref="ReadOnlyBuffer{T}"/> to a <see cref="Stream"/> asynchronously
         /// </summary>
-        /// <param name="buffer">The <see cref="ReadOnlyBuffer"/> to copy</param>
+        /// <param name="buffer">The <see cref="ReadOnlyBuffer{T}"/> to copy</param>
```
Comment (BrennanConroy) [inline src/Microsoft.AspNetCore.Sockets.Client.Http/Internal/StreamExtensions.cs]: T?

**Comment by BrennanConroy** [comment]
Don't forget to merge back to dev

**Pull request merged**